public/partials/search.html

```diff
@@ -34,11 +34,11 @@
                     </tr>
                     </thead>
                     <tbody>
-                    <tr ng-repeat="farmer in results.farmers" class="repeated-item" ng-click="selectedElement('farmer', farmer)">
-                        <td>{{ farmer.fa_jas_number }}</td>
-                        <td>{{ farmer.fa_first_name }}</td>
-                        <td>{{ farmer.fa_last_name }}</td>
-                        <td>{{ farmer.ad_address.pa_parish }}</td>
+                    <tr id="farmer-{{farmer._id}}" ng-repeat="farmer in results.farmers" class="repeated-item search-item" ng-click="highlightSearchItem($event);selectedElement('farmer', farmer);">
```

Comment:
No @nick22891.... assigning an id in this manner is a backward step in angular.  If you want to manipulate a particular item you can put a `ng-click` on the row and manipulate the farmer object itself.  Throughout this application we have examples.  Again.... try and build on what we have been doing.